### DISCUSSION TITLE
New Chat Window Design

### DISCUSSION THREAD

**Discussion by Isratja45** [chatwoot/chatwoot#7233] [General]
Wondering if you have plans to update your design for the chat widgets? It looks plain, kind of outdated compared to others like Intercom, Drift, crisp, chaport

**Comment by jochenstu** [comment]
FWIW I don't think it looks outdated :)

**Comment by iammattmartin** [comment]
Same, although it really needs to support a better level of customisation...

**Comment by glauberssoutodossantos** [comment]
@Isratja45 Hi,

I don't believe the widget is that out of date, but I think it could be better.

What I see that could use a big update is the interface for the agent. I see that it is very polluted, with a lot of loose buds everywhere.

I compare it a lot with WhatsApp web, which most people know.

Once I even did a study to try to fit chatwoot into a cleaner interface, more like WhatsApp.

I never took this study forward. If anyone wants to pick it up to implement.

![image](https://user-images.githubusercontent.com/99494469/215594568-acc812e3-3e9a-42e9-996a-226606da6a91.png)

**Comment by domaingood** [comment]
Maybe he talking about visitor Chat Windows.
